README.md

```diff
@@ -196,171 +196,12 @@ export function a11yAuditIf(contextSelector, axeOptions) {
-results of a component's audit, you can define a custom callback. The callback
-receives the results of the `a11yCheck` audit that is scoped to the component's
-element. Simply set it as `axeCallback` on the component in question:
-
-```javascript
-axeCallback(results) {
-  // do stuff with results
-}
-```
-
-##### Setting options for the audit
-
-As with testing, if you need to set custom auditing options for a component, you
-can do so easily. Simply set a value for the `axeOptions` property value:
-
-```javascript
-axeOptions: {
-  /* a11yCheck options */
-}
-```
-
-##### Turning the audit off
-
-Lastly, if you really find the audits to be cramping development, you can turn
-them off via a simple boolean switch:
-
-```javascript
-turnAuditOff: true;
-```
-
-#### Environment Options
-
-Each of the fine-grained component hooks above can instead be defined for ALL components inside of your application's `config/environment.js` file. Simply supply them in a `componentOptions` hash on the `ember-a11y-testing` property of `ENV`.
-
-```javascript
-ENV['ember-a11y-testing'] = {
-  componentOptions: {
-    turnAuditOff: false, // Change to true to disable the audit in development
-    axeCallback: defaultAxeCallback,
-    axeOptions: defaultAxeOptions,
-    visualNoiseLevel: 2,
-    axeViolationClassNames: ['alert-box', 'alert-box--a11y'],
-  },
-};
-```
-
-By example, to enable a specific rule and to set options for a specific check, you can:
-
-```javascript
-ENV['ember-a11y-testing']: {
-  componentOptions: {
-    axeOptions: {
-      rules: {
-        'region': { enabled: true } // enable region rule
-      },
-      checks: {
-        'color-contrast': { options: { noScroll: true } } // disable scrolling of color-contrast check
-      }
-    }
-  }
-}
-```
-
-#### Using Native aXe Global
-
-Ember A11y Testing injects the `axe` global during development and tests so that
-you can run accessibility audits while developing your application. It also
-provides some Ember-specific tools so that you can integrate accessibility
-checks into your workflow easily.
+- [**Lighthouse**](https://developers.google.com/web/tools/lighthouse) - an open-source, automated tool for improving the quality of web pages. You can run it against any web page, public or requiring authentication. It has audits for performance, accessibility, progressive web apps, SEO and more.
+  - Homepage: https://developers.google.com/web/tools/lighthouse
+- [**Sa11y**](https://ryersondmp.github.io/sa11y/) - an accessibility quality assurance tool that visually highlights common accessibility and usability issues. Geared towards content authors, Sa11y indicates errors or warnings at the source with a simple tooltip on how to fix.
+- [axe Chrome extension](https://www.deque.com/axe/browser-extensions/) - a free axe browser extension ideal for development teams to test web applications to help identify and resolve common accessibility issues.
 
```

Comment:
We should also reference Microsoft's Accessibility Insights extension, which is the one we primarily use. I'd probably list it first.
```suggestion
[**Accessibility Insights for Web**](https://accessibilityinsights.io/docs/en/web/overview) - Accessibility Insights for Web helps developers find and fix accessibility issues in web apps and sites. This browser extension for Chrome and the new Microsoft Edge runs on Windows, MacOS, and Linux computers.

```

Reply:
Ya, that's great to call out.

Reply:
Agree. Thanks for the callout, @drewlee!